history/settings/docker.py

```diff
@@ -3,14 +3,21 @@
 Docker settings file
 """
 # Local imports
-# Third-party imports
-import pymongo
 import os
-
-
+# Third-party imports
 # Local imports
-from base import *
+from history.settings.base import *
 
+MONGO_DB = os.environ.get('MONGO_DB', 'mongodb')
 replica_set = os.environ.get('MONGO_REPLICA_SET', None)
```

Comment:
Might change the replica_set variable to the Upper case format as well.

Reply:
done